website/source/docs/job-specification/reschedule.html.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: "docs"
+page_title: "reschedule Stanza - Job Specification"
+sidebar_current: "docs-job-specification-reschedule"
+description: |-
+  The "reschedule" stanza specifies the group's rescheduling strategy upon task failures.
+  The reschedule strategy can be configured with number of attempts and a time interval.
+  Nomad will attempt to reschedule failed allocations on to another node only after
+  any applicable [restarts](docs/job-specification/restart.html) have been tried.
+---
+
+# `reschedule` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> **reschedule**</code>
+    </td>
+    <td>
+      <code>job -> group -> **reschedule**</code>
+    </td>
+  </tr>
+</table>
+
+The `reschedule` stanza specifies the group's rescheduling strategy.
+It can be configured with number of attempts and a time interval. If
+omitted, a failed allocation will not be rescheduled on another node. If specified
+at the job level, the configuration will apply to all groups within the job. If multiple
+`reschedule` stanzas are specified, they are merged with the group stanza taking the
+highest precedence and then the job.
+
+Nomad will attempt to schedule the task on another node if any of its allocation statuses become
+"failed". It uses a penalty score to prefer nodes on which the task has not been previously run on.
```

Comment:
It prefers to create a replacement allocation on a node that hasn't previously been used.